ironfish/src/utils/blockchain.ts

```diff
@@ -40,6 +42,26 @@ export function getBlockRange(
   return { start, stop }
 }
 
+export const NATIVE_ASSET_VALUE = {
+  createdTransactionHash: GENESIS_BLOCK_PREVIOUS,
+  identifier: Asset.nativeIdentifier(),
+  metadata: Buffer.from('Native asset of Iron Fish blockchain', 'utf8'),
+  name: Buffer.from('$IRON', 'utf8'),
+  nonce: 0,
+  owner: Buffer.from('Iron Fish', 'utf8'),
+  supply: BigInt(0),
+}
+
+export async function getAssetById(
+  assetIdentifier: Buffer,
+  chain: Blockchain,
+): Promise<AssetsValue | undefined> {
+  if (Asset.nativeIdentifier().equals(assetIdentifier)) {
+    return NATIVE_ASSET_VALUE
+  }
+  return await chain.assets.get(assetIdentifier)
+}
```

Comment:
Agree with rohan, I expect this to be on block chain itself like all the other getter functions